src/components/ModalNavigation/ModalNavigation.tsx

```diff
@@ -2,15 +2,21 @@ import * as React from 'react'
 import './ModalNavigation.css'
 
 export type ModalNavigationProps = {
+  // * TITLE: modal's title
```

Comment:
Would you mind writing them in the [JSDoc](https://jsdoc.app/) format so any editor can pick them up later?
```suggestion
  /** The modal's title. */
```